hsm/README.md

```diff
@@ -0,0 +1,110 @@
+Using HSM
+=========
+
+HSM stands for Hierarchical Storage Management. m0hsm tool allows to
+create composite objects in Motr, write/read to/from them and move them
+between the tiers (pools). Here is how to use the tool:
+
+1. Set the following environment variables:
+
+   ```bash
+   export CLIENT_PROF_OPT="<0x7000000000000001:0x480>"   # profile id
+   export CLIENT_HA_ADDR="172.18.1.24@o2ib:12345:34:101" # ha-agent address
+   export CLIENT_LADDR="172.18.1.24@o2ib:12345:41:322"   # local address
+   export CLIENT_PROC_FID="<0x7200000000000001:0xdf>"    # process id
+   ```
+
+   Profile id of the cluster and ha-agent address on your client node can
+   be checked with `hctl status` command. As well as all addresses and
+   processes ids configured in the cluster. Consult with the cluster system
+   administrator about which of them you can use.
+
+2. Initialize the composite layout index:
+
+   ```Text
+   $ c0composite "$CLIENT_LADDR" "$CLIENT_HA_ADDR" "$CLIENT_PROF_OPT" "$CLIENT_PROC_FID"
```

Comment:
c0composite it is changed to m0composite now.

Reply:
Oh, good catch. Let me fix it...